console/v1/types_console_link.go

```diff
@@ -29,6 +39,8 @@ const (
 	HelpMenu ConsoleLinkLocation = "HelpMenu"
 	// UserMenu indicates that the link should appear in the user menu in the console.
 	UserMenu ConsoleLinkLocation = "UserMenu"
+	// ApplicationMenu indicates that the link should appear inside the applicationMenu of the console.
```

Comment:
We refer to "user menu" above. I would refer to it as "application menu" here and in other description (except when referencing the actual property name)

Reply:
Looks like this comment hasn't been addressed.